strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-pypi-layout-provider/src/test/java/org/carlspring/strongbox/util/PypiMetadataParserTest.java

```diff
@@ -42,22 +54,123 @@ public void testParseFileToPypiMetadataDto()
 
     @Test
     public void testParseFileWithEmptyValuesToPypiMetadataDto()
-            throws IllegalAccessException, IOException
     {
-        PypiMetadataParser pypiMetadataParser = new PypiMetadataParser();
-        PypiPackageInfo testDao = pypiMetadataParser.parseMetadataFile(new FileInputStream("src/test/resources/org.carlspring.strongbox.util/metadata_empty_values.xml"));
-
-        assertThat(testDao.getName()).isEmpty();
-        assertThat(testDao.getMetadataVersion()).isEmpty();
-        assertThat(testDao.getVersion()).isEmpty();
-        assertThat(testDao.getSummary()).isEmpty();
-        assertThat(testDao.getHomePage()).isEmpty();
-        assertThat(testDao.getAuthor()).isEmpty();
-        assertThat(testDao.getAuthorEmail()).isEmpty();
-        assertThat(testDao.getLicense()).isEmpty();
-        assertThat(testDao.getDescriptionContentType()).isEmpty();
-        assertThat(testDao.getDescription()).isEmpty();
-        assertThat(testDao.getPlatform()).isEmpty();
+        assertThrows(ConstraintViolationException.class, () -> {
+            pypiMetadataParser.parseMetadataFile(new FileInputStream("src/test/resources/org.carlspring.strongbox.util/PKG-INFO-empty-values"));
+        });
+    }
+
+    @Test
```

Comment:
Cosmetic: Could you please, re-indent your changes here according to our [coding convention](https://strongbox.github.io/developer-guide/coding-convention.html)? Thanks! :)

Reply:
@carlspring code is reformatted to follow coding convention. Please check again. Thanks!